tests/lib/snaps/snapctl-hooks/meta/hooks/configure

```diff
@@ -60,6 +60,20 @@ test_get_int_number() {
 	fi
 }
 
+test_get_nested() {
+    echo "Getting a map"
+    if ! output="$(snapctl get root)"; then
+		    echo "Expected snapctl get to be able to retrieve value just set"
+		    exit 1
+	  fi
+	  for expected_output in "\"key1\": \"a\"" "\"key2\": \"b\""; do
```

Comment:
Instead of the grep, could we do something like: `if [ "$output" != '{"key1": "a", "key2": "b"}' ]; then` here?

Reply:
We get multiline output here, and with tabs for indentation. I tried a couple of approaches here and eventually eneded up with single grep invocation with precise pattern for entire output, is that ok?